pallets/randomness/src/lib.rs

```diff
@@ -289,12 +308,10 @@ pub mod pallet {
 	#[pallet::hooks]
 	impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
 		fn on_initialize(_now: BlockNumberFor<T>) -> Weight {
-			// Do not set the output in the first block (genesis or runtime upgrade)
-			// because we do not have any input for author to sign
-			if NotFirstBlock::<T>::get().is_none() {
-				NotFirstBlock::<T>::put(());
-				LocalVrfOutput::<T>::put(Some(T::Hash::default()));
-				return T::DbWeight::get().reads_writes(1, 2);
+			// Do not verify default VrfOutput set in first block
+			if frame_system::Pallet::<T>::block_number().is_zero() {
```

Comment:
This will not work for networks who introduced VRF after the genesis

Reply:
This workaround is only necessary for our TS tests.

This PR **removes the randomness initialization** for running chains. Because we have already added randomness to all of our running networks, we do not need the initialization workaround anymore.

Reply:
@4meta5 Some of our pallets are used by other projects, and this could be the case for the randomness pallet in the future.

So I'm rather in favor of leaving an initialization if there is a way to implement it properly. Or else indicate very explicitly in the documentation of this pallet that it can't be added to an already running network, specifying why.

Reply:
Yeah I think I'm just going to abandon this PR and the ticket. I don't see any way to handle the edge case elegantly.